### ISSUE TITLE
Growing Object-Oriented Software, Guided by Tests

### ISSUE THREAD

**Issue by pavolzbell** [rubyslava/bookshelf#3]
@smolnar a ja by sme ju radi precitali, dalo by sa doniest zajtra na Rubyslavu?

**Comment by jsuchal** [comment]
@crutch nejaky update?

**Comment by crutch** [comment]
Mam to rozcitane - skusim si s tym viac pohnut.

**Comment by jsuchal** [comment]
@crutch nejaky posun?

**Comment by crutch** [comment]
Ano, uz som niekde za polovicou - ale je to na mna do elektricky miestami dost narocne, ale dobre citanie :) @srigi, Ty si to vlastne cital alebo na nu cakas?

**Comment by srigi** [comment]
Ja som v pohode s tym, ze je mate. Asi by som si teraz nevedel najst cas.

**Comment by ps-aux** [comment]
Ako to teraz vyzera s knizkou ? Mohol by som sa k nej v dohladnej dobe dostat ? :)

**Comment by crutch** [comment]
@ps-aux mohol. V ktorych castiach Bratislavy sa zvyknes vyskytovat?

**Comment by ps-aux** [comment]
Stare mesto, Dubravka a v pohode dobehnem aj niekde vokol.

**Comment by crutch** [comment]
Ja som z Karlovky. Ak by si sa vedel v pondelok vecer zastavit pri Centrume (Borska) tak Ti ju tam mozem priniest. Kontakt na mna: 0903 843 276.

**Comment by crutch** [comment]
este mi tak napadlo...@srigi si s tym ok? :)

**Comment by ps-aux** [comment]
`if (srigi.isOk())` tak zbehnem do Karlovky nejaky vecer

**Comment by srigi** [comment]
`srigi.isOk()  // TRUE`

**Comment by ps-aux** [comment]
Po novom roku kedy asi ? Ma to tak do 400 stran tak by som na to potreboval nejaky cas :)

**Comment by srigi** [comment]
Radovo mesiace po novom roku, nie skor ako v Marci.

**Comment by ps-aux** [comment]
Knizka je u mna.

**Comment by ps-aux** [comment]
Precitane a ready na posunutie dalej - Srigimu. Chcel som ti to dat na NodeSchool #2, potom na Rubyslave # 54 ale nebolo ta :)

**Comment by jsuchal** [comment]
@ps-aux das aj nejaku kratku recenziu?

**Comment by srigi** [comment]
@ps-aux bol som chory, sorry :). Dojdi na Webelement.

**Comment by ps-aux** [comment]
@jsuchal O recenziu sa mozem pokusit ale teraz som dost vytazeny takze neskor. @srigi Mozno dojdem pozriet na Webelement a donesiem ti tu knizku.

**Comment by srigi** [comment]
Kniha je teraz u mna.

**Comment by srigi** [comment]
Knihu som posunul @kmaraz, takze ju maju chalani zo Sli.do.
